CHANGELOG.rst

```diff
@@ -5,6 +5,8 @@
 New Features
 ------------
 
+* junitxml: Add `--junit-suite-name` option to specify root `<testsuite>` name for JUnit XML reports
```

Comment:
Oh, please also mention the original issue (#533) here (take a look at the other entries for see the syntax).